src/tests/test_integration/test_movies_models.py

```
@@ -7,9 +7,9 @@
 from database import MovieModel
 from database import (
     GenreModel,
-    ActorModel,
-    LanguageModel,
-    CountryModel
+    StarModel,
```

Ruff (0.11.9)

`database.StarModel` imported but unused

Remove unused import

(F401)